source/components/molecules/CharacterCard/CharacterCard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useContext } from "react";
+import { ThemeContext } from "styled-components/native";
+import Checkbox from "../../atoms/Checkbox";
+import { colorPalette } from "../../../styles/palette";
+
+import { Card, Section, Image, CardTitle, CardSubtitle } from "./styled";
+
+type CharacterCardType = {
+  icon: any;
+  title: string;
+  department: string;
+  jobTitle: string;
+  selected: boolean;
+  onCardClick: () => void;
+};
+const CharacterCard = (props: CharacterCardType): JSX.Element => {
+  const {
+    icon,
+    selected = false,
+    title,
+    department,
+    jobTitle,
+    onCardClick,
+  } = props;
+  const { colors } = useContext<any>(ThemeContext);
```

Comment:
Same with `any` here - there should be a specific type for `ThemeContext` right?